scripts/update_ci.sh

```diff
@@ -23,6 +23,7 @@ else
 fi
 
 pushd ../plugin_template
+pip install -r requirements.txt
 ./plugin-template --github $docs $plugin_name
 popd
```

Comment:
@mdellweg I've added the requirements here, because the workflow uses this script from plugin-template checkout, which will be up-to-date with what it needs.

That seems a resonable approach for handling plugin-template dependencies in general, wdyt?